src/com/cft/contactmerge/contact/NameProperty.java

```diff
@@ -0,0 +1,39 @@
+package com.cft.contactmerge.contact;
+
+import com.cft.contactmerge.AnswerType;
+
+public class NameProperty  implements IContactProperty<NameProperty>
+{
+    private FirstNameProperty firstName;
+    private LastNameProperty lastName;
+
+    public NameProperty(String lastName, String firstName)
+    {
+        this.lastName = new LastNameProperty(lastName);
+        this.firstName = new FirstNameProperty(firstName);
+    }
+
+    public AnswerType isMatch(NameProperty otherProperty) {
+        // TODO: Add code to return appropriate response
+        return null;
+    }
+
+    public NameProperty getValue()
+    {
+        return this;
+    }
+
+    public String toString() {
+        return lastName + ", " + firstName;
```

Comment:
Are we going to be using the comma as our main delimiter? Or do you think there is something better and less likely to be used as a part of a property value? Like tilde  or semicolon.

Reply:
No... This is just a standard way of displaying names. The other standard method is "<FirstName> <LastName>" but I prefer "<LastName>, <FirstName" because it works better in sorts and scans.